packages/databyss-ui/primitives/View/HoverView.js

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef } from 'react'
+import {
+  variant,
+  flexbox,
+  shadow,
+  layout,
+  compose,
+  border,
+} from 'styled-system'
+import { Grid } from '@databyss-org/ui/primitives'
+import styled from '../styled'
+import IS_NATIVE from '../../lib/isNative'
+
+const variants = variant({
```

Comment:
Instead of a view variant, let's move those style props into this component, as they are only used in one place rn